ci-operator/step-registry/ibmcloud/deprovision/bastionhost/ibmcloud-deprovision-bastionhost-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    "${IBMCLOUD_CLI}" login -a https://cloud.ibm.com -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+function checkCli() {
+  export IBMCLOUD_CLI=ibmcloud
+  export IBMCLOUD_HOME=/output
+  echo "check IBMCLOUD_CLI: ${IBMCLOUD_CLI}..."
+  command -v ${IBMCLOUD_CLI}
+  ${IBMCLOUD_CLI} --version
+  ${IBMCLOUD_CLI} plugin list
+}
+
+function getVarFromFile() {
+  local file val
+  file="$1"
+  if [[ ! -f "${file}" ]]; then
+    echo "Fail to find ${file}, abort." && exit 1
+  fi
+  val=$(cat ${file})
+  echo ${val}
+}
+
+function deleteBastion() {    
+  local bastionFile name ip
+  bastionFile="$1" rgName="$2"
+  name=$(yq-go r "${bastionFile}" 'bastionHost')
+  ip=$(yq-go r "${bastionFile}" 'publicIpAddress')
+  run_command "${IBMCLOUD_CLI} is instance-delete ${name} -f"
+  run_command "${IBMCLOUD_CLI} is ips --resource-group-name ${rgName} | grep -w ${ip} | cut -d ' ' -f1 | xargs ${IBMCLOUD_CLI} is ipd -f"
+}
+
+# ibmcloud should already be there
+checkCli
+
+# Disable exit on error for login
+set +o errexit
+login_success=false
+for _ in $(seq 5); do
+    if ibmcloud_login; then
+        login_success=true
+        break
+    fi
+    sleep 10
+done
+if [[ ${login_success} == false ]]; then
+    echo "ERROR: Failed to log into IBM Cloud CLI."
+    exit 1
+fi
+
+region="${LEASED_RESOURCE}"
+
+vpc_name=$(getVarFromFile "${SHARED_DIR}/ibmcloud_vpc_name")
+
+resource_group=$(getVarFromFile "${SHARED_DIR}/ibmcloud_resource_group")
+echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpc_name}"
+
+${IBMCLOUD_CLI} target -g ${resource_group} -r ${region}
+
+bastion_info_yaml="${SHARED_DIR}/bastion-info.yaml"
+if [ -f "${bastion_info_yaml}" ]; then
```

Comment:
In prow ci, when calling the step by users, that means, the `bastion-info.yaml` must be existing, no need to check if it is existing or not.